### ISSUE TITLE
Render Airflow TaskGroups with Clusters

### ISSUE THREAD

**Issue by nathadfield** [feluelle/airflow-diagrams#12]
Airflow DAGs that utilise TaskGroups would be better represented if we were to make use of [Clusters](https://diagrams.mingrammer.com/docs/guides/cluster)

**Comment by feluelle** [comment]
The problem I have with this feature is that retrieving the information from airflow whether it belongs to a taskgroup or not is unsafe. We cannot retrieve it from the rest api directly as it currently only is a [UI concept](https://airflow.apache.org/docs/apache-airflow/stable/concepts/dags.html#concepts-taskgroups). I have to parse the task_id 😟.

But we can give it a try.

**Comment by nathadfield** [comment]
Yes, that's true and presumably gets trickier if there are multiple levels of TaskGroups?  I know there is a desire to make TaskGroups more than just a UI construct so perhaps this will be easier in the future.
